dali/operators/geometry/coord_flip.cc

```diff
@@ -12,37 +12,37 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "dali/operators/coord/coord_flip.h"
+#include "dali/operators/geometry/coord_flip.h"
 
 namespace dali {
 
 DALI_SCHEMA(CoordFlip)
     .DocStr(
-        R"code(Transforms coordinates so that they are flipped (point reflected) with respect
-to a center point.)code")
+        R"code(Transforms vectors or points by flipping (reflecting) their coordinates with
+respect to a given center.)code")
```

Comment:
We're not reflecting with respect to a _point_ - we're reflecting with respect to multiple subspaces.